src/service/qumlBulkService.js

```diff
@@ -0,0 +1,153 @@
+const fs = require("fs");
+const fetch = require("node-fetch");
+const { v4: uuidv4 } = require("uuid");
+const KafkaService = require("../helpers/kafkaUtil");
+const logger = require('sb_logger_util_v2');
+const loggerService = require('./loggerService');
+const responseCode = messageUtils.RESPONSE_CODE;
+const programMessages = messageUtils.PROGRAM;
+const errorCodes = messageUtils.ERRORCODES;
+const envVariables = require("../envVariables");
+const rspObj = {};
+
+const bulkUpload = async (req, res) => {
+    const logObject = {
+        traceId : req.headers['x-request-id'] || '',
+        message : programMessages.CONTENT_PUBLISH.INFO
+      }
+  let errorArray = [];
+  let pId = uuidv4();
+  let successArray = [];
+  let qumlData;
+  const fileType = req.files.File.mimetype;
```

Comment:
shouldn't it be `req.body.files.File.mimeType` ? our request will be always under the `body` right?

Reply:
No Since it is in form data it will come as files